helm/templates/nginx-network-policy.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: nginx
+spec:
+  podSelector:
+    matchLabels:
+      app: nginx
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - from:
+    - ipBlock:
+        cidr: 0.0.0.0/0
+    - podSelector:
+        matchLabels: {}
```

Comment:
what is the meaning of empty pod selector?

Reply:
I believe that means any pod. It could be omitted altogether.
We can specify the pod label here to restrict this further.
Which pod is sending traffic to `nginx`?

Reply:
I think it means "no pod", not "any pod", and it is meaningless if ipblock cidr 0.0.0.0 is specified

Regarding pod sending traffic to nginx, for now you can leave it to ipBlock cidr 0.0.0.0 without specifying pod